src/bitcoin/HexaConfig.ts

```diff
@@ -269,6 +272,31 @@ class HexaConfig {
       this.NETWORK = bitcoinJS.networks.testnet;
     }
   };
+
+  public connectToOwnNode =  async () => {
+    let ownNodes:any = await AsyncStorage.getItem('OwnNodes')
+
+    if(ownNodes){
+      ownNodes = JSON.parse(ownNodes);
+      const ownNodeURL = ownNodes.urls[ownNodes.active];
+      if(ownNodeURL){
+        this.ESPLORA_API_ENDPOINTS = {
+          ...this.ESPLORA_API_ENDPOINTS,
+           MAINNET: {
```

Comment:
@Parsh Is the assumption here that own node will only work for MAINNET and not on TESTNET?
The testnet account can continue to use Hexa Node/wrapper. However for dev env will we be able to test own node functionality and if so will MAINNET object have testnet end points in that case?

Reply:
Yes, I don't think anyone is going to spin up a Testnet node w/ a Testnet indexer just to make their Testnet account work. Also, the standby indexer that Varun has is mainnet one, so we'll be testing this on staging, hence we ignored the testnet configuration altogether.